examples/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "basic-google-map",
+  "version": "1.0.0",
+  "private": true,
+  "description": "A sample Google Map setup with the MapProvider component.",
+  "main": "index.js",
+  "scripts": {
+    "clean-examples": "rm -rf public .parcel-cache",
+    "start:map": "PARCEL_AUTOINSTALL=false parcel serve ./basic-google-map/index.html --dist-dir public --port 1234",
+    "start:map-markers": "PARCEL_AUTOINSTALL=false parcel serve ./google-map-with-markers/index.html --dist-dir public --port 1234"
+  },
+  "author": "",
```

Comment:
remove or add value

Reply:
Do we want a value here?

Reply:
Discussed: We will add the author to the main package but not to the examples :)

Reply:
Ok
```suggestion
```